src/figuresClasses.ts

```diff
@@ -1,19 +1,11 @@
-export interface Figure {
+export interface Figure {}
 
-}
-
-export class Triangle {
-
-}
-
-export class Circle {
+export class Triangle implements Figure {}
 
-}
-
-export class Rectangle {
-
-}
+export class Circle implements Figure {}
 
-export function getInfo(figure) {
+export class Rectangle implements Figure {}
 
+export function getInfo(figure): string {
```

Comment:
The `getInfo` function should return a string containing information about the figure, such as its color and area, as indicated by the test cases. Consider implementing logic to extract and format these details from the `figure` object.